lib/config.js

```diff
@@ -16,35 +16,49 @@ const Config = {
   schema: {
     autocomplete: {
       title: 'Enable Autocomplete',
+      includeTitle: false,
```

Comment:
I had to [read the code](https://github.com/atom/settings-view/blob/197f962fec680e1b9b1c7c8f5fd162a99731d959/lib/settings-panel.coffee#L48-L69) to fix this issue.